lib/Api/Api.php

```diff
@@ -163,11 +163,11 @@ public function get($id)
      *
      * @return array|mixed
      */
-    public function getList($filter = '', $start = 0, $limit = 0, $orderBy = '', $orderByDir = 'ASC', $publishedOnly = false)
+    public function getList($search = '', $start = 0, $limit = 0, $orderBy = '', $orderByDir = 'ASC', $publishedOnly = false)
     {
         $parameters = array();
 
-        $args = array('filter', 'start', 'limit', 'orderBy', 'orderByDir', 'publishedOnly');
+        $args = array('search', 'start', 'limit', 'orderBy', 'orderByDir', 'publishedOnly');
 
         foreach ($args as $arg) {
             if (!empty($$arg)) {
```

Comment:
Not related to this PR but noticed this line has a bug in it thanks to GitHub cutting the diff off here LOL